target/linux/ath79/image/generic.mk

```diff
@@ -1732,9 +1732,12 @@ define Device/nec_wg800hp
 endef
 TARGET_DEVICES += nec_wg800hp
 
-define Device/netgear_ex6400_ex7300
+define Device/netgear_ex7300
   $(Device/netgear_generic)
   SOC := qca9558
+  DEVICE_MODEL := EX7300
+  DEVICE_ALT0_VENDOR := NETGEAR
+  DEVICE_ALT0_MODEL := EX6400
   UIMAGE_MAGIC := 0x27051956
```

Comment:
`0x27051956` is the default u-boot magic. Please remove this line.

Reply:
Done